packages/warning/tsconfig.json

```
@@ -2,7 +2,8 @@
 	"extends": "../../tsconfig.base.json",
 	"compilerOptions": {
 		"rootDir": "src",
-		"declarationDir": "build-types"
+		"declarationDir": "build-types",
+		"types": [ "gutenberg-env" ]
```

How about using `node` for this? Its typedefs [seem to include](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/20c9b607bf37bdc372faa0f1b079ac6d964ad287/types/node/globals.d.ts#L789) `process.env.NODE_ENV`.

```suggestion
		"types": [ "node" ]
```

This would allow us to drop the `gutenberg-env` typings, and the typings dir altogether (for now -- YAGNI).

---

I think we want to avoid the node types and only add as much as is necessary, hence the custom typing. Node will tweak the environment slightly, so things like `setTimeout`/`clearTimeout` are slightly different. It includes a a bunch of things that shouldn't be present, e.g. `__dirname`.

---

Okay, makes sense.